src/index.js

```diff
@@ -96,3 +96,7 @@ on(displayedEvent, () => {
         started = true;
     }
 });
+
+const rootView = function () {
+    return getRootView() || android.foregroundActivity.getWindow().getDecorView().getRootView();
```

Comment:
The name `android` conflicts with the global `android` namespace.

This function isn't cross platform.
I don't know if `getRootView()` can return a falsy value on `iOS` but if if can, this will crash.